waku/waku_api/rest/doc/relayapi_auto_content_topic.yaml

```diff
@@ -0,0 +1,25 @@
+# /relay/v1/auto/messages/{contentTopic}:  # Note the plural in messages
+get: # get_waku_v2_relay_v1_auto_messages
+  summary: Get the latest messages on the polled topic
+  description: Get a list of messages that were received on a subscribed Content topic after the last time this method was called.
+  operationId: getMessagesByTopic
+  tags:
+    - relay
+  parameters:
+    - in: path
+      name: contentTopic   # Note the name is the same as in the path
+      required: true
+      schema:
+        type: string
+      description: The user ID
+  responses:
+    '200':
+      description: The latest messages on the polled topic.
+      content:
+        application/json:
+          schema:
+            $ref: './schemas/apitypes.yaml#/RelayGetMessagesResponse'
+    '4XX':
```

Comment:
Would 4XX be returned in case the node is not subscribed to that particular contentTopic?
Hope we have an error message being returned for all error responses indicating the specific error that occured.